### DISCUSSION TITLE
Check sums for bin downloads of micropython?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [micropython/micropython#16101] [General]
Check sums for bin downloads of micropython?  Say I want to validate 'ESP32_GENERIC_C6-20240602-v1.24.0.bin' after wget bin file?

**Comment by sosi-deadeye** [comment]
There are no hash checksums available, so you can't check them.

**Comment by Jibun-no-Kage** [comment]
Bummer, well thought to ask, since I could not find them.
